programs/autocrat/src/instructions/autocrat/finalize_proposal.rs

```diff
@@ -66,6 +66,8 @@ pub fn handler(ctx: Context<FinalizeProposal>) -> Result<()> {
     let treasury_seeds = &[dao_pubkey.as_ref(), &[dao.treasury_pda_bump]];
     let signer = &[&treasury_seeds[..]];
 
+    // floats feel especially scary here, I think we should stick to integers
```

Comment:
I can use integers. I thought it was especially not scary though since proposals are on the order of 5% difference, so any tiny rounding error would not matter? Lmk your thoughts though.

Reply:
I'll just use integers everywhere, nbd

Reply:
No one ever got fired for using integers